src/boxed.rs

```diff
@@ -205,6 +205,14 @@ macro_rules! glib_boxed_wrapper {
             }
         }
 
+        #[doc(hidden)]
+        impl $crate::translate::FromGlibPtrBorrow<*const $ffi_name> for $name {
+            #[inline]
+            unsafe fn from_glib_borrow(ptr: *const $ffi_name) -> Self {
+                $name($crate::translate::from_glib_borrow(ptr as usize as *mut $ffi_name))
```

Comment:
Why the cast via `usize`? That shouldn't be needed at all, just the cast to a mutable pointer should be needed

Reply:
Also everywhere else